src/xrpld/app/misc/CredentialHelpers.cpp

```diff
@@ -225,6 +226,31 @@ makeSorted(STArray const& in)
     return out;
 }
 
+NotTEC
+checkArray(STArray const& in, unsigned maxSize)
```

Comment:
nit: the variable `in` should be renamed because it's confusing when there's a separate variable in the function called `ins`

Reply:
renamed